utils/anaInfo.py

```diff
@@ -12,8 +12,8 @@
 
 import string
 
-#: If the chisquare value of the fit to DAC vs ADC is above this value, an exception will be raised
-dacScanFitChisquareMax = 100.
+#: If the chisquare value divided by number of degrees of freedom of the fit to DAC vs ADC is above this value, an exception will be raised
+dacScanFitChiSqOverNDFMax = 15.
```

Comment:
i see this change, but no corresponding change elsewhere, is this expected?

Reply:
specifically, i see that you changed `dacScanFitChisquareMax` -> `dacScanFitChiSqOverNDFMax` in `anaInfo.py`, but not where `dacScanFitChisquareMax` was presumably used in the actual fitting routine code

Reply:
yes, I apparently made the change and then unintentionally reverted it, sorry, it is fixed now, and I have rebased